lemur/certificates/cli.py

```diff
@@ -641,11 +641,13 @@ def revoke(path, reason, commit):
 
     print("[+] Starting certificate revocation.")
 
-    with open(path, "r") as f:
-        args = [[x, commit, reason] for x in f.readlines()[2:]]
+    if reason not in CRLReason.__members__:
+        reason = CRLReason.unspecified.name
+    comments = {"comments": message, "crl_reason": reason}
 
-    with multiprocessing.Pool(processes=3) as pool:
-        pool.starmap(worker, args)
```

Comment:
This method was not working when used with multiprocessing (tried using ThreadPoolExecutor too), the error it gives is.
`No application found. Either work inside a view function or push an application context. See http://flask-sqlalchemy.pocoo.org/contexts/.`
Even with app_context, it complained when tried to modify certificate object later in code. So moving it to sequential processing. It can be revisited if actively used and performance improvement is needed.